test/suite/test_gc01.py

```diff
@@ -87,6 +73,26 @@ def check(self, check_value, uri, nrows, read_ts):
         session.rollback_transaction()
         self.assertEqual(count, nrows)
 
+    def check_gc_stats(self):
+        c = self.session.open_cursor(
+            'statistics:', None, 'statistics=(fast)')
```

Comment:
I think these lines can be merged into one? Does it matter that we specify `statistics=(fast)` here? Using `None` gets us the connection configuration which is `all` so anything we're looking for should be there and performance is not an issue.

Reply:
No, it does not. Lines merged and code updated to `c = self.session.open_cursor('statistics:')`.